.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXXXX
```

Comment:
Pas sûr qu'ils vont utiliser JIRA pour le coup 🤔

Reply:
C'est pour nous ça, mais on pourra le retirer dès qu'on transmet le projet ! ;)

Reply:
Je me demande si real / virtural device c'est bon

Reply:
Je pense que c'est mieux de mettre ça ? 
`Made sure my feature is working on different browsers (Chrome, Firefox, Safari).`

Reply:
Ah oui, c'est mieux 👍